model-server/src/main/kotlin/org/modelix/model/server/handlers/ContentExplorer.kt

```diff
@@ -0,0 +1,87 @@
+package org.modelix.model.server.handlers
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.server.thymeleaf.*
+import org.modelix.model.ModelFacade
+import org.modelix.model.api.*
+import org.modelix.model.client.IModelClient
+import org.modelix.model.lazy.CLVersion
+import org.modelix.model.lazy.RepositoryId
+import org.modelix.model.metameta.MetaModelBranch
+
+class ContentExplorer(private val client: IModelClient) {
+
+    private val knownRepositoryIds: Set<HistoryHandler.RepositoryAndBranch>
```

Comment:
There is now a `RepositoriesManager` that provides this information, but I just realised that it isn't yet backward compatible to this legacy way of storing repository meta information (aka "info" repository).